src/components/SecondaryPanes/Breakpoints/index.js

```diff
@@ -79,14 +80,23 @@ class Breakpoints extends Component<Props> {
     return [
       ...breakpointSources.map(({ source, breakpoints, i }) => {
         const path = getDisplayPath(source, sources);
+        const sortedBreakpoints = sortBy(
+          breakpoints,
+          ({ selectedLocation }) => [
+            selectedLocation.line,
+            (selectedLocation.column || 0).toString().length,
+            selectedLocation.column || 0
```

Comment:
why is this so complicated?

is this shared with some other logic that should be in a util?